mlrun/runtimes/constants.py

```diff
@@ -85,6 +85,13 @@ def terminal_states():
     def non_terminal_states():
         return list(set(RunStates.all()) - set(RunStates.terminal_states()))
 
+    @staticmethod
+    def k8s_non_terminal_states():
+        return [
+            RunStates.pending,
+            RunStates.running,
+        ]
+
```

Comment:
What is the idea behind this list ? what's special to those states (in contrast to `non_terminal_states`)

Reply:
talked offline, removed.